build-libs/redirects.js

```diff
@@ -339,8 +339,7 @@ async function redirectsConfig() {
 	const dotIoRedirects = await buildDotIoRedirects()
 	const devPortalRedirects = await buildDevPortalRedirects()
 	const { simpleRedirects, globRedirects } = splitRedirectsByType([
-		buildBetaProductOptInRedirect('waypoint', ['docs', 'commands', 'plugins']),
-		buildBetaProductOptInRedirect('vault', ['docs', 'api-docs']),
+		...buildBetaProductOptInRedirects(),
```

Comment:
Oh wow! I didn't even know this was something we should have been updating previously. Great call to automate as much as possible 👍🏻